libs/libc/string/lib_strncasecmp.c

```diff
@@ -50,4 +50,9 @@ int strncasecmp(const char *cs, const char *ct, size_t nb)
 
   return result;
 }
+
+int strncasecmp_l(FAR const char *cs, FAR const char *ct, size_t nb, locale_t l)
```

Comment:
@xiaoxiang781216 Now we have an nxstyle error :rofl:

Reply:
Done.